tests/Dfc.CourseDirectory.WebV2.Tests/TestViews/SharedViews/PagerTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using Microsoft.AspNetCore.Mvc;
+using Xunit;
+
+namespace Dfc.CourseDirectory.WebV2.Tests.TestViews.SharedViews
```

Comment:
Put this in `Dfc.CourseDirectory.WebV2.Tests.ViewComponentTests` namespace

Reply:
Done